### PR TITLE
Implement wp db check

### LINKED ISSUE #3273: wp db check
We have optimize and repair but no check.

### PR DESCRIPTION
**Pull request by szepeviktor**
Closes #3273

### CODE REVIEW

File: features/db.feature [hunk 1]
```diff
@@ -21,6 +21,9 @@ Feature: Perform database operations
     When I try the previous command again
     Then the return code should be 1
 
+    When I run `wp db check`
+    Then STDOUT should not be empty
```
Comment (danielbachhuber) [inline features/db.feature]: Can you improve this test to:

```
Then STDOUT should be:
```

And verify the output is what's expected?

Comment (szepeviktor) [inline features/db.feature]: I was encouraged by the other two tests.

Does it seem OK?
```
wp_cli_test.wp_commentmeta                         OK
wp_cli_test.wp_comments                            OK
wp_cli_test.wp_links                               OK
wp_cli_test.wp_options                             OK
wp_cli_test.wp_postmeta                            OK
wp_cli_test.wp_posts                               OK
wp_cli_test.wp_term_relationships                  OK
wp_cli_test.wp_term_taxonomy                       OK
wp_cli_test.wp_termmeta                            OK
wp_cli_test.wp_terms                               OK
wp_cli_test.wp_usermeta                            OK
wp_cli_test.wp_users                               OK
```

Comment (danielbachhuber) [inline features/db.feature]: > Does this seem OK?

Yep.

Comment (szepeviktor) [inline features/db.feature]: Do you have security-lazy users??

secure-wplogin

It is free: https://github.com/szepeviktor/wordpress-plugin-construction/blob/master/mu-keepass-button/keepass-button.php

Comment (danielbachhuber) [inline features/db.feature]: @szepeviktor Did you mean to post ^ ?

Comment (szepeviktor) [inline features/db.feature]: @danielbachhuber Yes.

Comment (danielbachhuber) [inline features/db.feature]: Ok. That's off-topic for the pull request, and not really appropriate for the project. I'm going to delete the comment. Please keep conversations on-topic.

Comment (szepeviktor) [inline features/db.feature]: OK :( Let's be serious.

**Comment by szepeviktor** [comment]
@danielbachhuber Do you have an idea why PHP7 east the output?
https://travis-ci.org/wp-cli/wp-cli/builds/151661997

**Comment by danielbachhuber** [comment]
> Do you have an idea why PHP7 easts the output?

Nope :(

**Comment by szepeviktor** [comment]
Do we have another another test that tests STDOUT?

**Comment by szepeviktor** [comment]
My fault.

**Comment by szepeviktor** [comment]
I have no clue...
https://github.com/wp-cli/wp-cli/blob/master/php/utils.php#L434

**Comment by szepeviktor** [comment]
When I run `wp db check` it lists all the tables to STDOUT.

**Comment by szepeviktor** [comment]
@danielbachhuber Could you run wp db check yourself?

**Comment by ernilambar** [comment]
I tried this PR. In Terminal, I got expected output. But in Behat test, table output is not there, only `Success: Database checked.`.

**Comment by szepeviktor** [comment]
@danielbachhuber Are you a Behat expert?

**Comment by danielbachhuber** [comment]
> Are you a Behat expert?

Not necessarily. Sorry I haven't taken the time to debug this yet. I'll try to make time to do so in the near future.

**Comment by szepeviktor** [comment]
I am trying to find the solution before you.

**Comment by danielbachhuber** [comment]
Closing in favor of #3332

**Comment by szepeviktor** [comment]
https://github.com/wp-cli/wp-cli/commit/3ce2409f8de98292df14d27cccaddcaac3e1a9a8 is not fair.

Behat has a problem, I don't know what it is. It swallows stdout.

-- 
+36204242498
Ezen a készüléken sok az elütés. 
Elnézést!

**Comment by danielbachhuber** [comment]
> Behat has a problem, I don't know what it is. It swallows stdout.

Hm, not locally according to d6870493f6d26a9b0c66c84697025bc31dee3c81

**Pull request closed without merging**